spec/features/display_marked_as_blank_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'spec_helper'
+
+describe "display marked as blank" do
+  let(:deed_type) { DeedType::PAGE_MARKED_BLANK }
+
+  it 'shows pages marked blank in main Activity Feed page' do
+    # Build out factories
+    user = create(:user)
+    collection = create(:collection, owner_user_id: user.id)
+    work = create(:work, owner_user_id: user.id, collection_id: collection.id)
+    page1 = create(:page, work_id: work.id)
+    deed = create(:deed, {
+      deed_type: deed_type,
+      page_id: page1.id,
+      work_id: work.id,
+      collection_id: collection.id,
+      user_id: user.id
+      })
+
+    # Visit page
+    visit 'deed/list'
+    expect(page).to have_content('Page Marked Blank')
+
+    # Tear down Factories
+    Deed.destroy(deed.id)
+    Page.destroy(page1.id)
+    Work.destroy(work.id)
+    Collection.destroy(collection.id)
+    User.destroy(user.id)
+  end
```

Comment:
Since we're using fixtures and not doing a database cleanup after each `it` statement, I've found that I need to generate and destroy my factory data manually each time in order to keep my tests clean. If there is a better way, I'd love to implement that.